utils/batch.ts

```diff
@@ -95,6 +96,21 @@ export const removeEmptyKeysFromObject = (obj: any) => {
   return obj;
 };
 
+export const convertToOsStandard = (obj: NFTMetadataInput["attributes"]) => {
+  const keys = Object.keys(obj || {});
+  const values = Object.values(obj || {});
+  const attributes = [];
+
+  for (let i = 0; i < keys.length; i++) {
+    attributes.push({
+      trait_type: keys[i],
+      value: values[i],
+    });
+  }
```

Comment:
could be replaced with this

```suggestion
  const attributes = Object.entries(obj || {}).map(([trait_type, value]) => ({
    trait_type,
    value,
  }));
```

Reply:
well that's miles easier, thanks